import-export-cli/cmd/bundle.go

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
Shall we add the commands and usage of this feature to the file in [1]?

[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/resources/README.html

Reply:
Fixed with a3a6fff